jenkins/bootstrap.py

```diff
@@ -782,6 +787,10 @@ def parse_args(arguments=None):
     parser.add_argument(
         '--service-account',
         help='Activate and use path/to/service-account.json if set.')
+    parser.add_argument(
+        '--ssh',
+        nargs='?', const=1, default=0,
```

Comment:
store_true is less code, but for the person using this from a shell I kind of like being able to distinguish between "I don't care", "yes" and "no"